bot/resources/evergreen/py_topics.yaml

```diff
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Was Python your first programming language? If not, what was?
+    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
```

Comment:
Isn't `web dev` included with `backend`? You cannot do frontend with it...

Reply:
Technically `web dev` could just be `frontend`, but for clarity I think I'll just replace `web dev` with `frontend`